### PR TITLE
layout: Switch `.len()` comparison to `is_empty()` in `components/layout_2020/query.rs`.

### PR DESCRIPTION
**Pull request by DanielFrantes**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #___ (GitHub issue number if applicable)

- [ ] There are tests for these changes OR
- [x] These changes do not require tests because dont chenge app logic

### CODE REVIEW

File: components/layout_2020/query.rs [hunk 1]
```diff
@@ -741,7 +741,7 @@ fn rendered_text_collection_steps<'dom>(
                     items.push(InnerOrOuterTextItem::Text(String::from(" ")));
                 };
 
-                if transformed_text.len() > 0 {
+                if !transformed_text.is_empty() == true {
```
Comment (mrobinson) [inline components/layout_2020/query.rs:744]: You shouldn't need to compare a boolean expression against true.

```suggestion
                if !transformed_text.is_empty() {
```

**Review by mrobinson — CHANGES_REQUESTED** [review]

File: components/layout_2020/query.rs (hunk 1 above)
Comment (danik292) [reply components/layout_2020/query.rs:744]: Thank I forgot about this.🙃

Comment (danik292) [reply components/layout_2020/query.rs]: Should be ok now

File: components/layout_2020/query.rs [hunk 2]
```diff
@@ -741,7 +741,7 @@ fn rendered_text_collection_steps<'dom>(
                     items.push(InnerOrOuterTextItem::Text(String::from(" ")));
                 };
 
-                if transformed_text.len() > 0 {
+                if !transformed_text.is_empty(){
```
Comment (mrobinson) [inline components/layout_2020/query.rs:744]:
```suggestion
                if !transformed_text.is_empty() {
```

**Review by mrobinson — APPROVED** [review]

**Pull request merged**